Tests/SwiftParserTest/translated/GuardTests.swift

```diff
@@ -9,18 +9,23 @@ final class GuardTests: XCTestCase {
       func noConditionNoElse() {
         guard {} 1️⃣
       }
-      func noCondition() {
-        guard 2️⃣else {} 
-      }3️⃣
       """,
       diagnostics: [
-        // TODO: Old parser expected error on line 2: missing condition in 'guard' statement
-        // TODO: Old parser expected error on line 2: expected 'else' after 'guard' condition
-        DiagnosticSpec(locationMarker: "1️⃣", message: "expected 'else' and body in 'guard' statement"),
-        // TODO: Old parser expected error on line 5: missing condition in 'guard' statement
-        DiagnosticSpec(locationMarker: "2️⃣", message: "expected expression in 'guard' statement"),
+        DiagnosticSpec(message: "expected 'else' and body in 'guard' statement"),
       ]
     )
   }
 
+  func testGuard2() {
+    AssertParse(
+      """
+      func noCondition() {
+        guard 1️⃣else {}
+      }
+      """,
+      diagnostics: [
+        DiagnosticSpec(message: "expected conditions in 'guard' statement"),
```

Comment:
I know we allow "conditions" but IMO "condition" makes more sense here, ie. we expect at least one *condition*, not necessarily multiple.